pkg/testhelpers/fixtures.go

```diff
@@ -0,0 +1,19 @@
+package testhelpers
+
+import (
+	"io/ioutil"
+	"path/filepath"
+)
+
+func Fixture(name string) string {
```

Comment:
```suggestion
func MustFixture(name string) string {
```
I think that's the convention